lib/child_process.js

```diff
@@ -47,7 +48,6 @@ const {
 
 const {
   convertToValidSignal,
-  createDeferredPromise,
```

Comment:
If this isn't being used, you can remove it entirely, right?

Reply:
yes

Reply:
I removed it because it was not in use